.github/ISSUE_TEMPLATE/product-listing-request.md

```diff
@@ -8,27 +8,20 @@ assignees: ''
 
 ---
 
+## Instructions:
+1. Make a copy of the [New Product Listing Spreadsheet](https://docs.google.com/spreadsheets/d/1KugFl4qv9plCtFXfRgd5Ip0bXpAiZo6Yqm8CnWEBOUg/edit#gid=1512129648) and give the engineering team edit permissions
+2. Fill the copy New Product Listing Spreadsheet accordingly and link it to the **New Product Listing Spreadsheet** section below
+3. Fill the **Onchain data** and **SDK data** sections for each of the new product
+4. Delete this **Instructions** section before submitting the Product Listing Request issue
+
 # Product Listing Request
 
+**New Product Listing Spreadsheet**:
+
 ## Onchain data
```

Comment:
Could we combine the `## Onchain data` and `## SDK data` headings and just make it `## <product name>`? I noticed that it reads really annoying when copy-pasting data if there are a few products being added at the same time. I was really anxious I would read/copy something wrong because the issue content is so similar. Differentiating with a simple productname as a title would make it a lot more readable.